images/pegasus/Dockerfile

```diff
@@ -0,0 +1,142 @@
+ARG BASE_APP_IMAGE
+
+# hadolint ignore=DL3006
+FROM ${BASE_APP_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+# see: https://github.com/AppImage/AppImageKit/wiki/FUSE#docker
+ENV APPIMAGE_EXTRACT_AND_RUN=1
+
+# Install prereqs
+RUN \
+    echo "**** Install Prereqs (Mesa/Vulkan/Fuse/QT/Misc) ****" && \
+		apt-get update && \
+		apt-get install -y software-properties-common gpg-agent wget p7zip-full libvulkan1 mesa-vulkan-drivers libglu1-mesa-dev qtbase5-dev qt5-qmake ffmpeg x11-xserver-utils libdbus-1-3 libgtk-3-0 libegl1 libsdl2-2.0-0 && \
+	# Cleanup \
+		apt-get autoremove -y && \
+		rm -rf /var/lib/apt/lists/*
+
+# Install RetroArch
+RUN \
+    echo "**** Install RetroArch ****" && \
+		apt-get update && \
+		apt-get install -y --no-install-recommends software-properties-common gpg-agent libusb-1.0-0 libglu1-mesa libaio1 libaio-dev && \
```

Comment:
Could this be `from: ghcr.io/games-on-whales/retroarch` instead?

Reply:
Not a bad idea. Could we also do the same for the `es-ds` image too?